docs/sphinx/developers/building-bioformats.txt

```diff
@@ -114,11 +114,40 @@ and JAR dependencies (stored in the Maven repository in
 :file:`~/.m2/repository`) on the build path, depending on which projects are
 currently open.
 
-We recommend using Eclipse 4.3 (Kepler), specifically -
+We recommend using Eclipse 4.3 (Kepler) or later, specifically -
 "Eclipse IDE for Java developers". It comes with m2e installed
 (http://eclipse.org/downloads/compare.php?release=kepler).
 
-You can then import the Bio-Formats source by choosing
+Bio-Formats can be included as a dependency in any Maven project. The 
```

Comment:
If we were to restructure content, I would suggest this could migrated to http://www.openmicroscopy.org/site/support/bio-formats5.2/developers/java-library.html which is about using Bio-Formats. This should be done in sync with @hflynn who is going to update this page though.